runtime/major_gc.c

```diff
@@ -542,7 +542,7 @@ static void mark_ephe_aux (struct mark_stack *stk, intnat *work,
       ephes_to_check = &Field(v,CAML_EPHE_LINK_OFFSET);
       return;
     }
-  } else {  /* a simily weak pointer or an already alive data */
+  } else {  /* a similarly weak pointer or an already alive data */
```

Comment:
Was that really a typo for `similarly` (and not a frenchism for `simili`)?

Reply:
I'd convinced myself it was close enough, but perhaps @bobot can clarify?

Reply:
Either similarly or possibly "likewise", looking at the context

Reply:
OK, thanks - I'll change with a direct push shortly

Reply:
How does https://github.com/dra27/ocaml/commit/6da02508d5f33d871e88606d549ce12a1c20d4b2 look? (not yet pushed!)

Reply:
Ohh, I see! I'm not sure there's an exact idiomatic translation for just using simili, but how about: `/* No data (like a weak pointer) or data already marked as alive. */`